pkg/client/clientset/versioned/fake/register.go

```diff
@@ -23,14 +23,14 @@ var localSchemeBuilder = runtime.SchemeBuilder{
 // AddToScheme adds all types of this clientset into the given scheme. This allows composition
 // of clientsets, like in:
 //
-//	import (
-//	  "k8s.io/client-go/kubernetes"
-//	  clientsetscheme "k8s.io/client-go/kubernetes/scheme"
-//	  aggregatorclientsetscheme "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/scheme"
-//	)
+//   import (
```

Comment:
Alignment diffs?

Reply:
That is the revert of https://github.com/openshift-knative/serverless-operator/pull/2024. And what I meant with, golang versions might have different formatting. https://github.com/openshift-knative/serverless-operator/pull/2024 did format them with 1.19, this reverts it back to the style of 1.18.